src/root/port.d

```diff
@@ -52,40 +42,21 @@ extern (C++) struct Port
         static __gshared bool yl2x_supported = false;
         static __gshared bool yl2xp1_supported = false;
     }
-    static __gshared real snan;
-    static this()
-    {
-        /*
-         * Use a payload which is different from the machine NaN,
-         * so that uninitialised variables can be
-         * detected even if exceptions are disabled.
-         */
-        ushort* us = cast(ushort*)&snan;
-        us[0] = 0;
-        us[1] = 0;
-        us[2] = 0;
-        us[3] = 0xA000;
-        us[4] = 0x7FFF;
-    }
-
-    static bool isNan(double r)
-    {
-        return !(r == r);
-    }
 
-    static real sqrt(real x)
+    static void yl2x(real_t* x, real_t* y, real_t* res)
     {
-        return .sqrt(x);
-    }
-
-    static real fmodl(real a, real b)
-    {
-        return a % b;
+        version(DigitalMars)
+            *res = core.math.yl2x(*x, *y);
+        else
+            assert(0);
     }
 
-    static real fequal(real a, real b)
```

Comment:
A `real` should be big enough to store the bool. I went with a `bool` nevertheless. ;)